.vitepress/theme/components/landing/1. hero-section/HeroSection.vue

```diff
@@ -5,14 +5,14 @@ import HeroDiagram from './HeroDiagram.vue'
 <template>
   <div class="hero">
     <div class="container">
-      <!-- ViteConf Replay Button -->
+      <!-- ViteConf 2025 Button -->
       <a
-          href="https://viteconf.org/24/replay?utm=vite"
+          href="https://viteconf.org/?utm=vite"
           class="hero__pill"
           target="_blank"
       >
         <img src="/viteconf.svg" alt="Viteconf logo" width="20" height="20" />
-        <span>ViteConf 2024 のトーク</span>
+        <span>ViteConf 2025 のトーク</span>
```

Comment:
英語版は "Talks" がなくなっているので、こちらも削除でお願いします 🙏 
```suggestion
        <span>ViteConf 2025</span>
```